extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -273,7 +273,12 @@ To make payment via our integration, you need to set the `makePaymentRequest` cu
 - `makePaymentRequest` must contain a unique payment `reference` value. The reference value cannot be duplicated in any commercetools payment and it's a required field by Adyen. The extension module uses the `reference` value to set the payment key, later it acts as a unique link between commercetools payment and Adyen payment(`merchantReference`). `Reference` may only contain alphanumeric characters, underscores and hyphens and must have a minimum length of 2 characters and a maximum length of 80 characters.
 - `payment.amountPlanned` can not be changed if there is a `makePayment` interface interaction present in the commercetools payment object. The `amount` value in `makePaymentRequest` custom field must have the same value as `payment.amountPlanned`. This ensures eventual payment amount manipulations (i.e.: when [my-payments](https://docs.commercetools.com/http-api-projects-me-payments#my-payments) are used) for already initiated payment.
 
-Here's an example of the value of the `makePaymentRequest` custom field, with generated component data from Adyen Web Components for a credit card payment:
+**Important**
+
+In this integration document our Adyen payment request examples are trimmed to minimum. Depending on your requirements you might want to include other Adyen parameters such as [add risk management fields](https://docs.adyen.com/risk-management/configure-standard-risk-rules/required-risk-field-reference), [activate 3D Secure 2](https://docs.adyen.com/online-payments/3d-secure/native-3ds2/web-component#make-a-payment) or [allow recurring payments](https://docs.adyen.com/payment-methods/cards/web-component#create-a-token).
+Please find all the possible parameters in the `Web Components` section of the desired payment method listed in the navigation [here](https://docs.adyen.com/payment-methods).
+
+Here's an example of the value of the `makePaymentRequest` custom field for 3D Secure credit cart payment , with generated component data from Adyen Web Components for a credit card payment:
```

Comment:
I think we need `3D Secure 2` - 2 is important, as we are telling to the Adyen - our app is ready for 3ds2 handling. I am pretty sure, if it is not provided - it would force 3ds1 (redirect) if card's issuer requires it.

Also we could get rid of this space before `, with`